packages/app/src/app/components/CodeEditor/CodeMirror/index.js

```diff
@@ -54,6 +55,7 @@ class CodemirrorEditor extends React.Component<Props, State> implements Editor {
     this.settings = props.settings;
 
     this.codeSandboxListener = this.setupCodeSandboxListener();
+    this.resizeEditor = debounce(this.resizeEditor, 30);
```

Comment:
curious: what's the significance of this exact number?

Reply:
It's a random number really, but I've seen this number in other codebases as well when debouncing a resize. I think makes sure that it happens as fast as possible after an animation.